programs/witness_node/main.cpp

```diff
@@ -106,18 +106,24 @@ static void load_config_file( const fc::path& config_ini_path, const bpo::option
 
    // get the basic options
    bpo::store(bpo::parse_config_file<char>(config_ini_path.preferred_string().c_str(),
-              unique_options, true), options);
+                                           unique_options, true), options);
+}
 
+static void load_logging_config_file
+(
+   const fc::path& config_ini_path
+)
```

Comment:
style - parameters on same line, if they fit, makes it easier to distinguish this as a function declaration

Reply:
fixed